rdrf/rdrf/templates/proms/proms_completed.html

```diff
@@ -1,7 +1,20 @@
-<html>
-    <body>
-
-    <h1>You have completed the survey - thank you!</h1>
-    </body>
-
-</html>
+{% load static from staticfiles %}
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <title>Thank you</title>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet" href="{% static 'bootstrap-3.3.6/css/bootstrap.min.css' %}">
+   </head>
+   <body>
```

Comment:
Please discuss any arbritary ( non-ticketed changes first) - we need to create separate tickets for all work and prioritise